.github/workflows/docker-latest.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image for latest release
+
+on:
+  push:
+    tags:
+      - release
+
+env:
+  TEST_TAG: zeek:latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - name: Get Version
+        id: version
+        run: echo "::set-output name=RELEASE_VERSION::$(cat VERSION)"
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile
+          load: true
+          tags: ${{ env.TEST_TAG }}
+      - name: Run tests
+        uses: plexsystems/container-structure-test-action@v0.2.0
+        with:
+            image: ${{ env.TEST_TAG }}
+            config: docker/container-structure-test.yaml
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile
+          push: true
+          tags: |
+            zeekurity/zeek:latest
```

Comment:
Here I was wondering about the relationship to `TEST_TAG`.

Reply:
`TEST_TAG` is just for testing. This step build the image again (with a warm cache so the build should be a noop), doesn't `load` it but instead performs a `push` under the given `tags`.